docs/user/reference/environment-variables.rst

```diff
@@ -16,13 +16,14 @@ All :doc:`build processes </builds>` have the following environment variables au
 .. envvar:: READTHEDOCS_LANGUAGE
 
     The locale name, or the identifier for the locale, for the project being built.
-    This value comes from the project's configured language.
+    This value comes from the project's configured language,
+    normalized to be lowercase and use a dash as a separator instead of an underscore.
```

Comment:
Don't think we need to change this paragraph, the language should be shown as lowercase everywhere already.

Reply:
No notes for those who expected to be in the format e.g. pt_BR? Asking this because I faced this issue in Sphinx doc translation and then I landed on this page with this out-of-date info.